server/app/assets/javascripts/enumerator.ts

```diff
@@ -119,6 +121,36 @@ function removeExistingEnumeratorField(event: Event) {
 
   // Add the hidden deleted entity input to the page.
   enumeratorFieldDiv.appendChild(deletedEntityInput)
+
+  setFocusAfterEnumeratorRemoval()
+}
+
+/**
+ * Set focus after an enumerator entity is removed. If no other entries are
+ * present, will set to the add button, otherwise will set to the last entity's
+ * remove button. Setting focus is important for a11y (e.g. for keyboard only
+ * or screenreader users).
+ */
+function setFocusAfterEnumeratorRemoval() {
+  const deleteButtons = document.querySelectorAll(
+    '.cf-enumerator-field:not(.hidden) .cf-enumerator-delete-button',
+  )
+  if (deleteButtons.length == 0) {
```

Comment:
suggestion (blocking): use `===` instead of `==` in most cases for JS. [style guide entry](https://google.github.io/styleguide/jsguide.html#features-equality-checks)

```suggestion
  if (deleteButtons.length === 0) {
```

Reply:
Thanks! Didn't know this about javascript.